transformer_engine/pytorch/cpp_extensions/gemm.py

```diff
@@ -458,3 +464,83 @@ def fp8_grouped_gemm(
     )
 
     return out, gelu_input
+
+
+def fp8_grouped_gemm_single_output(
```

Comment:
This new function is testsed in `tests/pytorch/test_numerics.py::test_grouped_linear_accuracy`, which consists of two `fp8_grouped_gemm_single_output` and one `fp8_grouped_gemm`.